src/components/routerView.vue

```diff
@@ -59,7 +62,7 @@
       // eslint-disable-next-line @typescript-eslint/no-unused-expressions, no-unused-expressions
       route.params
 
-      return componentUtil(component, () => props(route.params))
+      return componentUtil(component, () => props)
```

Comment:
My guess is since the props are called within the store this is no longer reactive to params changing. Is that true?

Reply:
we have [this unit test](https://github.com/kitbagjs/router/blob/main/src/components/routerView.browser.spec.ts#L335) that I think is asserting that props are still reactive

Reply:
Hmm right, not sure how though lol.

Reply:
Wanna try removing line 63 and see if the tests still pass now that the prop store is using `reactive`?